src/chi/components/tables/tables.scss

```diff
@@ -2,8 +2,233 @@
+          margin-left: 0.5rem
+        }
+      }
+
+      &.-ascending {
+        div::after {
+          opacity: 1;
+        }
+      }
+
+      &.-descending {
+        div::after {
+          opacity: 1;
+          transform: rotate(180deg);
+        }
+      }
+    }
+
+    & td {
+      border-bottom: 0.0625rem solid set-color(grey, 20);
+      padding: 1rem;
+      padding-bottom: 0.9375rem;
+      vertical-align: middle;
+    }
+
+    & tbody {
+      tr {
+        &.-row--success {
+          background-color: set-color(green, 10);
+        }
+
+        &.-row--warning {
+          background-color: set-color(yellow, 10);
+        }
+
+        &.-row--danger {
+          background-color: set-color(red, 10);
+        }
+
+        &.-row--info {
+          background-color: set-color(blue, 10);
+        }
+
+        &[disabled] {
+          color: rgba(set-color(grey, 90), 0.4);
+        }
+      }
+    }
+
+    &.-hover {
+      tbody {
+        tr {
+          &:hover,
+          &.-hover {
+            background-color: set-color(grey, 10);
+          }
+        }
+      }
+    }
+
+    &.-striped {
+      tbody {
+        tr:nth-child(odd) {
+          background-color: set-color(grey, 10);
+        }
+      }
+    }
+
+    &.-borderless {
+      &::before {
+        border: 0;
+      }
+
+      th {
+        border-bottom: 0.0625rem solid set-color(grey, 30);
+      }
+
+      td {
+        border: 0;
+        padding: 1rem;
+      }
+    }
+
+    &.-bordered {
+      &::before {
+        border: 0.0625rem solid set-color(grey, 30);
+        border-bottom-width: 0;
+        border-left-width: 0;
+      }
+
+      th {
+        border-bottom: 0;
+      }
+
+      th {
+        border: 0.0625rem solid set-color(grey, 30);
+        border-right-width: 0;
+        border-style: solid;
+
+        &:last-of-type {
+          border-right-width: 0.0625rem;
+        }
+      }
+
+      td {
+        border: 0.0625rem solid set-color(grey, 30);
+        border-right-width: 0;
+        border-top-width: 0;
+
+        &:last-of-type {
+          border-right-width: 0.0625rem;
+        }
+      }
+    }
+
+    &.-small {
+      tr {
+        height: 2rem;
+      }
+
+      th {
+        padding: .5rem 1rem .4375rem .9375rem
+      }
+
+      td {
+        padding-bottom: 0.4375rem;
+      }
+
+      &.-borderless {
+        td {
+          padding: 0.5rem 1rem;
+        }
+      }
+
+      &.-bordered {
+        td {
+          padding: 0.5rem 1rem 0.4375rem 0.9375rem;
+        }
+      }
+    }
+
+    &.-large {
+      tr {
+        height: 4rem;
+      }
+
+      th {
+        font-size: $text;
+      }
+
+      td {
+        padding: 1.5rem 1rem;
+      }
+
+      td {
+        padding-bottom: 1.4375rem;
+      }
+
+      &.-borderless {
+        td {
+          padding: 1.5rem 1rem;
+        }
+      }
+
+      &.-bordered {
+        td {
+          padding: 1.5rem 1rem 1.4375rem 0.9375rem;
+        }
+      }
+    }
+
+    &.-advanced {
```

Comment:
@davidojedacl can we call this something more specific? -fixed-header ? Any suggestions @jllr ?
Also, need to fix the double border on the th

Reply:
@davidojedacl @mattnickles -fixed--header sounds good to me